ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh

```diff
@@ -685,6 +685,38 @@ function upgrade() {
         clear_upgrade
         check_upgrade_status "${cluster_src_ver}"
     fi
+
+    if check_ota_case_enabled "OCP-56083"; then
```

Comment:
From the following case steps, sounds like mainly the following steps:
1. unset channel
2. trigger upgrade
3. check clusterversion message

From my understanding, 1 should be treated as some pre-ugprade actions, 3 should be some post-upgrade actions. If that, 1 should be placed in `cucushift-ota-preupgrade`, 3 should be placed in `cucushift-ota-preupgrade`, 2 already covered in this `cucushift-upgrade-toimage`.

Reply:
OK, Thanks @jianlinliu, i will update the changes accordingly